### PR TITLE
Saas security: Modified layout to added and remove several fields. Fi…

### PR DESCRIPTION
**Pull request by zhiwangprisma**
…x icon and wording.

## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: link to the issue

## Description
A few sentences describing the overall goals of the pull request's commits.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/PrismaSaasSecurity/pack_metadata.json [hunk 1]
```diff
@@ -1,5 +1,5 @@
 {
-    "name": "Saas Security (Prisma)",
+    "name": "Saas Security (Beta)",
```
Comment (bakatzir) [inline Packs/PrismaSaasSecurity/pack_metadata.json:2]: If you want an integration to be Beta, I suggest specifying it in the integration yml file.
in the display field and using the `bet: true` property.
sse this file for example: https://github.com/demisto/content/blob/master/Packs/AWS-AccessAnalyzer/Integrations/AWS-AccessAnalyzer/AWS-AccessAnalyzer.yml

Comment (zhiwangprisma) [reply Packs/PrismaSaasSecurity/pack_metadata.json:2]: updated pr accordingly

File: Packs/PrismaSaasSecurity/IncidentFields/incidentfield-Saas_Security_Incident_ID.json [hunk 2]
```diff
@@ -3,7 +3,7 @@
     "version": -1,
     "modified": "2021-10-27T22:56:13.134377204Z",
     "fromVersion": "6.0.0",
-    "name": "Saas Security Incident Id",
+    "name": "Saas Security Incident ID",
```
Comment (tomer-pan) [inline Packs/PrismaSaasSecurity/IncidentFields/incidentfield-Saas_Security_Incident_ID.json:6]: @ilaner @bakatzir I'm not sure - is it okay to change fields names?

**Review by ilaner — APPROVED** [review]
Great job

**Pull request merged**